server.js

```diff
@@ -32,22 +32,22 @@ let cfgConfig = {};
 let cfgFile = path.resolve(nconf.get('configfile') || nconf.get('configFile') || './.config.js');
 let cfgReason;
 const cfgFileExt = cfgFile.substr((cfgFile.lastIndexOf('.') + 1));
-if (cfgFile && fs.existsSync(cfgFile)) {
+if (cfgFile && fs.existsSync(cfgFile)) {       // eslint-disable-line no-sync
```

Comment:
you can disable no-sync for the entire file by putting a comment at the top of the file.  That probably makes sense in thi case.

Reply:
Yep! Will do for this file and a few others that have synchronous 'fs' calls.